css/ampdoc.css

```diff
@@ -76,6 +76,23 @@ html.i-amphtml-fie > body {
   overflow: visible !important;
 }
 
+/**
+ * iOS jitters when body overflows. See b/149328043.
+ */
+html.i-amphtml-singledoc.i-amphtml-iframed:not(.i-amphtml-ios-embed) > body {
+  overflow-x: hidden !important;
+  overflow-y: visible !important;
+}
+
+/**
+ * iOS jitters when body overflows. See b/149328043. This essentially cancels
+ * `height: auto` from above.
+ */
+html.i-amphtml-singledoc.i-amphtml-iframed:not(.i-amphtml-ios-embed):not([amp4ads]) > body {
+  min-height: calc(100vh - var(--i-amphtml-padding-top, 0px));
```

Comment:
Why was this added?

Reply:
We use to have something like this before for ios-embed mode. Here it's needed again, b/c the `body` loses the `overflowY: visible`. W/o the minimal height, any `position:absolute` element may get cut off.